clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClient.java

```diff
@@ -592,4 +593,29 @@ static ZeebeClientCloudBuilderStep1 newCloudClientBuilder() {
    */
   @ExperimentalApi("https://github.com/camunda/camunda/issues/20596")
   UserTaskQuery newUserTaskQuery();
+
+  /**
+   * Executes a search request to query decision definitions.
+   *
+   * <pre>
+   * long decisionDefinitionKey = ...;
+   *
+   * zeebeClient
+   *  .newDecisionDefinitionQuery()
+   *  .filter((f) -> f.key(decisionDefinitionKey))
```

Comment:
💭  Following the `DecisionDefinitionFilter` definition, I believe this would be decisionKey on example ? 

```suggestion
   *  .filter((f) -> f.decisionKey(decisionDefinitionKey))
```